src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/feedbackSessionDetailsPanel.tag

```diff
@@ -30,7 +30,7 @@
                 <div class="form-group">
                     <label class="col-sm-2 control-label">Closing time:</label>
                     <div class="col-sm-10">
-                        <p class="form-control-static">${feedbackSession.endTimeString}</p>
+                        <p class="form-control-static" id="end-time">${feedbackSession.endTimeString}</p>
```

Comment:
I'd prefer if the UTC time is embedded as a data attribute here in [JavaScript-friendly format](https://stackoverflow.com/questions/10286204/the-right-json-date-format?answertab=votes#tab-top), e.g.
```
<p class="form-control-static" id="end-time" data-end-time="2012-04-23T18:25:43.511Z">Tue, 24 Apr 2012, 2:25 AM UTC+8</p>
```
This would make the data independent of the way it's displayed. Also, the time zone info wouldn't have to be processed from JavaScript. Instead, the date object can be created simply with:
```
const endDate = new Date($(END_TIME).data('endTime'));
```